lib/Ix/Processor/JMAP.pm

```diff
@@ -16,6 +17,27 @@ use Ix::JMAP::SentenceCollection;
 
 with 'Ix::Processor';
 
+=head1 OVERVIEW
+
+This is a Moose role designed to make JMAP processors (the kind of processor
+used by an L<Ix::App::JMAP>) easy to write. It automatically adds method
+handlers for the standard JMAP methods (/get, /set, /changes, and maybe /query
+and /queryChanges) which are built from your L<Ix::DBIC::Result> classes. See
+the documentation there for more details.
+
+=cut
+
+=method handler_for($method_name)
+
+Implementations are required to implement this. It gets passed a method name
+(like 'Core/echo') and must return a coderef, which is later called with the
+parameters C<$self>, an C<Ix::Context> object, and the method's arguments as a
```

Comment:
This whole block should be rewritten I think. True, you need a handler for, but it could just be a stub that returns undef. And then the standard Ix provided jmap handlers will be used.

The only reason to return a coderef is if you have additional handlers for methods not provided by ix.

Reply:
I have rewritten this and added some examples, thanks.